ansible/roles/stack-sunbird/templates/inbound.env

```diff
@@ -53,14 +53,16 @@ REDIS_PASS={{sunbird_redis_pass | default('')}}
 REDIS_PORT={{sunbird_redis_port | default(6379)}}
 REDIS_DB_INDEX={{redis_db_index_uci | default('7')}}
 
-#Azure Config
-AZURE_BLOB_STORE_CONTAINER={{sunbird_azure_uci_container_name | default('uci-' + env )}}
-AZURE_BLOB_STORE_ACCOUNT_KEY={{sunbird_private_storage_account_name}}
-AZURE_BLOB_STORE_ACCOUNT_NAME={{sunbird_private_storage_account_key}}
-SELECTED_FILE_CDN=azure
-
 #Netcore
 NETCORE_WHATSAPP_AUTH_TOKEN={{uci_netcore_whatsapp_token}}
 NETCORE_WHATSAPP_SOURCE={{uci_netcore_whatsapp_source}}
 NETCORE_WHATSAPP_URI={{uci_netcore_whatsapp_uri | default('https://waapi.pepipost.com/api/v2/')}}
 
+#Sunbird CDN Configuration
+SUNBIRD_CLOUD_MEDIA_STORAGE_TYPE={{cloud_service_provider}}
+SUNBIRD_CLOUD_MEDIA_STORAGE_KEY={{cloud_public_storage_accountname}}
```

Comment:
@surabhi-mahawar  I think the storage account is private. Please check and confirm. If yes, change the accountname and secret vars accordingly.

Reply:
@santhosh-tg 
I have changed the storage account variables to private. 
Please review it.